types/stripe-v3/stripe-v3-tests.ts

```diff
@@ -42,6 +42,7 @@ describe("Stripe elements", () => {
             value: {
                 postalCode: '94110',
             },
+            disableLink: true
```

Comment:
```suggestion
            disableLink: true,
```
Add the trailing comma here please.

Trailing commas reduce the diff for the next PR's code review